### PR TITLE
Merge changes from beta 19.5.0.1 and editorialized release notes

### PR DESCRIPTION
**Pull request by mokagio**
Includes:

- The 19.4.1 hotfix, but that diff is empty. See #18188.
- A removal of some log messages that could have crashed the app [#18191](https://github.com/wordpress-mobile/WordPress-iOS/pull/18191) by @ScoutHarris 
 
Notice that the diff doesn't include the usual set of translations. The reason is explained in https://github.com/wordpress-mobile/WordPress-iOS/pull/18199.

When I noticed that some `.strings` were not committed, I decided not to cancel the build and to still open this PR because: 

- Being a beta, we can live with missing localizations (even thought the sooner we test them the better)
- We have the editorialized release notes to get into `trunk` and the beta lane commits on the release branch itself, and since that's protected, I couldn't rewrite the history and try again after #18199 is merged.

### CODE REVIEW

File: WordPress/Resources/ar.lproj/InfoPlist.strings [hunk 1]
```diff
@@ -1,5 +1,6 @@
...
-NSLocationWhenInUseUsageDescription = "WordPress would like to add your location to posts on sites where you have enabled geotagging.";
-NSCameraUsageDescription = "لالتقاط صور أو مقاطع الفيديو لاستخدامها في مقالاتك.";
-NSPhotoLibraryUsageDescription = "لإضافة صور أو مقاطع فيديو إلى مقالاتك.";
-NSMicrophoneUsageDescription = "بالنسبة إلى مقاطع الفيديو المراد إضافة صوت إليها.";
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<!--Warning: Auto-generated file, do not edit.-->
+<plist version="1.0">
+  <dict/>
+</plist>
```
Comment (mokagio) [inline WordPress/Resources/ar.lproj/InfoPlist.strings:6]: See https://github.com/wordpress-mobile/WordPress-iOS/pull/18061 for why this and all the other `.strings` in this diff changed in such a way.

**Review by mokagio — COMMENTED** [review]
- [x] Updates to the localization files (`.strings` and Fastlane metadata) automatically pulled for the new translations that have already been approved in GlotPress – With the caveat from the PR description
- [x] Version update in `.xcconfig`
- [x] `AppStoreStrings.po` changed with editorialized release notes
- [x] Diffs from the PRs that made it into this beta

File: WordPress/Resources/de.lproj/InfoPlist.strings [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+    <key>NSCameraUsageDescription</key>
+    <string>Um Fotos oder Videos für deine Beiträge aufzunehmen.</string>
+    <key>NSLocationUsageDescription</key>
+    <string>Darf WordPress zu Beiträgen auf Websites, auf denen du Geotagging aktiviert hast, deinen Standort hinzufügen?</string>
+    <key>NSLocationWhenInUseUsageDescription</key>
+    <string>Darf WordPress zu Beiträgen auf Websites, auf denen du Geotagging aktiviert hast, deinen Standort hinzufügen?</string>
+    <key>NSMicrophoneUsageDescription</key>
+    <string>Aktiviere den Mikrofonzugriff, um in deinen Videos Ton aufzeichnen zu können.</string>
+    <key>NSPhotoLibraryUsageDescription</key>
+    <string>Um deinen Beiträgen Fotos oder Videos hinzuzufügen.</string>
```
Comment (AliSoftware) [inline WordPress/Resources/de.lproj/InfoPlist.strings:15]: Nice to see some translations slowly coming in for those 🎉

File: WordPress/WordPressDraftActionExtension/de.lproj/InfoPlist.strings [hunk 3]
```diff
@@ -1,2 +1,11 @@
...
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<!--Warning: Auto-generated file, do not edit.-->
+<plist version="1.0">
+  <dict>
+    <key>CFBundleDisplayName</key>
+    <string>Als Entwurf speichern</string>
+    <key>CFBundleName</key>
+    <string>Als Entwurf speichern</string>
+  </dict>
```
Comment (AliSoftware) [inline WordPress/WordPressDraftActionExtension/de.lproj/InfoPlist.strings:10]: Likewise 🎉

File: WordPress/WordPressIntents/de.lproj/Sites.strings [hunk 4]
```diff
@@ -1,12 +1,19 @@
...
+    <string>Website</string>
+    <key>cyajMn</key>
+    <string>Website</string>
+    <key>gpCwrM</key>
+    <string>Website auswählen</string>
+    <key>s4dJhx</key>
+    <string>Nur um sicherzugehen, du willst „${site}“?</string>
+    <key>tVvJ9c</key>
+    <string>Website-Intent auswählen</string>
+  </dict>
```
Comment (AliSoftware) [inline WordPress/WordPressIntents/de.lproj/Sites.strings:18]: 🎉

**Review by AliSoftware — APPROVED** [review]
- [X] Changes from bugfix PRs which landed in the release branch
    - #18162/#18186, #18191
 - [X] Version bump in `.xcconfig` files
 - [X] Latest app and metadata translations pulled from Glotpress
    - As you mentioned, no update to `Localizable.strings` but that's because of the issue you encountered with them not being properly committed with the new actions; will follow up on that in #18199 
    - `InfoPlist.strings` and `Sites.strings` files updated with latest translations, thanks to the new actions and process now properly handling them 🎉 
 - [X] Editorialized Release Notes for WP+JP are in

**Pull request merged**